### PR TITLE
error estimation from multiple runs

### LINKED ISSUE #67: Error estimation from multiple runs
When N > 1 runs are averaged, the output should also show an estimate of the error of the mean (1 sigma) IMHO that should be **the best estimate of the mean** so reults are < x > +/- S
i.e.  
s = sqrt (1/(N-1) sum (x_i - < x_i >))
S = s / sqrt(N)

### PR DESCRIPTION
**Pull request by grzanka**
Fixes #67

### CODE REVIEW

**Comment by grzanka** [comment]
Right now mean error estimation is calculated always and alway included in plots. I thought if it should be like that or it should be done on demand. @nbassler any suggestions ?

**Comment by nbassler** [comment]
--no-errorbars ?

Nice work!

**Comment by grzanka** [comment]
@nbassler do you prefer by default to have or not to have error bars ?

File: pymchelper/detector.py [hunk 1]
```diff
@@ -98,9 +90,27 @@ def average_with_nan(self, other_detectors):
         l = [det.data for det in other_detectors]
         l.append(self.data)
         self.data = np.nanmean(l, axis=0)
+        self.error = np.nanvar(l, axis=0, ddof=1)
```
Comment (grzanka) [inline pymchelper/detector.py]: Should be np.nanstd, variance, not stderr

**Comment by nbassler** [comment]
@grzanka default (when nothing specified) should be `stderr` error bars.

**Pull request merged**